### PR TITLE
Add(Script/Cosmetic) Add super awesome most needed party time script

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed:**

-  TC was missing partying blood elves and I know that most users will say this is most needed in the core and is the best addition ever.
- Party on Garth!
- Party on Wayne!

**Tests performed:**

Script does build
SQL does insert
Blood elves do party

**Known issues and TODO list:**

I don't think anything is wrong but you never know :)

### CODE REVIEW

**Comment by acidmanifesto** [comment]
Screaming goats when?

**Comment by malcrom** [comment]
I will need to work on the screaming goats :)

File: src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                    me->HandleEmoteCommand(EMOTE_ONESHOT_LAUGH);
+                    _events.ScheduleEvent(EVENT_RANDOM_ACTION, 13s, 20s);
+                    break;
+                case 4:
+                    me->LoadEquipment(urand(1, 2));
+                    me->HandleEmoteCommand(EMOTE_ONESHOT_EAT_NO_SHEATHE);
+                    _events.ScheduleEvent(EVENT_REMOVE_EQUIPMENT, 4s);
+                    break;
+                case 5:
+                    me->SetUInt32Value(UNIT_NPC_EMOTESTATE, EMOTE_STATE_DANCE);
```
Comment (Shauren) [inline src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp:171]: Do not set UNIT_NPC_EMOTESTATE manually by directly using SetUInt32Value, use `SetEmoteState(EMOTE_STATE_DANCE)`

File: src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                    break;
+                }
+                break;
+            }
+            case EVENT_REMOVE_EQUIPMENT:
+                me->LoadEquipment(0, true);
+                _events.ScheduleEvent(EVENT_RANDOM_ACTION, 10s, 20s);
+                break;
+            case EVENT_STOP_DANCING:
+                me->SetUInt32Value(UNIT_NPC_EMOTESTATE, EMOTE_ONESHOT_NONE);
```
Comment (Shauren) [inline src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp:187]: same here, SetEmoteState

File: src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp (hunk 1 above)
Comment (malcrom) [reply src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp:171]: Will fix after I eat.

Comment (malcrom) [reply src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp]: all 4 fixed

File: src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp (hunk 2 above)
Comment (malcrom) [reply src/server/scripts/EasternKingdoms/zone_eversong_woods.cpp]: all 4 fixed

**Comment by malcrom** [comment]
While waiting for approval: https://www.youtube.com/watch?v=S0L_n68bznE

**Pull request merged**